configs/test/bigquery/datasets.yaml

```diff
@@ -209,25 +209,19 @@ resources:
             type: STRING
           - name: weight
             type: FLOAT
-  - name: fuzz-strategy-experiments-table
+  - name: fuzz-strategy-probability-table
     type: gcp-types/bigquery-v2:tables
     properties:
       datasetId: $(ref.main-dataset.datasetReference.datasetId)
       tableReference:
-        tableId: fuzz_strategy_experiments
+        tableId: fuzz_strategy_probability
```

Comment:
Happen to know if this table still exists in the internal project from before? We'll likely need to delete it if so.

Reply:
It doesn't, I'm pretty sure we deleted it earlier